### PR TITLE
Bot file decomposition and translations

### PR DESCRIPTION
**Pull request by korostast**
Fully translated bot
- Main features:
  - All text are translatable
  - Monotonicity function
  - Bot.py decomposed
- Fixes:
  - Meme and language changing buttons work
- Other:
  - Logically correct answers to some functions instead of Boolean or unclear answers

### CODE REVIEW

File: source/core/handling_msg.py [hunk 1]
```diff
@@ -1,200 +1,372 @@
...
-                await resize_image(latex_picture, resized_image)
+        @dispatcher.message_handler(commands=["graph"])
+        @rate_limit(limit=2)
+        async def graph(message: types.Message):
+            """Draw graph, save it as image and send to the user."""
+            if message.text == '/graph':
+                await Handler.mongo.go_graph(message)
+            else:
+                # await hmsg.send_graph(message)
+                asyncio.create_task(Handler.send_graph(message, ))
```
Comment (RaEzhov) [inline source/core/handling_msg.py:94]:
```suggestion
                await Handler.send_graph(message)
```

File: source/math/analyse_patterns.json [hunk 2]
```diff
@@ -261,5 +261,22 @@
...
+    },
+    "keywords": [
+      "the",
+      "function",
+      "monotonicity",
+      "monotonic",
+      "type"
+    ]
   }
 }
```
Comment (RaEzhov) [inline source/math/analyse_patterns.json:282]:
```suggestion
}

```
No newline, maybe it needs

**Review by RaEzhov — APPROVED** [review]

File: source/math/analyse_patterns.json (hunk 2 above)
Comment (Korostast) [reply source/math/analyse_patterns.json:282]: No, really, Github is a little dumb

File: source/core/handling_msg.py (hunk 1 above)
Comment (Korostast) [reply source/core/handling_msg.py:94]: Ah, yes, lovely asynchrony

File: source/core/handling_msg.py [hunk 3]
```diff
@@ -1,200 +1,371 @@
 """
 In this module we process events related to bot (such as messages, requests)
 """
+import asyncio
```
Comment (RaEzhov) [inline source/core/handling_msg.py:4]:
```suggestion
```

**Review by RaEzhov — APPROVED** [review]

**Pull request merged**